src/odemis/driver/actuator.py

```diff
@@ -1916,7 +1916,7 @@ class RotationActuator(model.Actuator):
     """
 
     def __init__(self, name, role, dependencies, axis_name, cycle=2 * math.pi,
-                 ref_start=None, inverted=None, **kwargs):
+                 ref_start=None, ref_on_move_count=5, inverted=None, **kwargs):
```

Comment:
I think it is a good idea to add this parameter, but I do not particularly like the name as it is hard to understand what it should mean. It should indicate the frequency of the referencing procedure - not based on a time, but on the number of moves executed. So I suggest something like: `ref_freq_move` or better just `ref_frequency`.

Reply:
I renamed it to `ref_frequency`